pkgs/by-name/ma/maltego/package.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, fetchzip
+, jre
+, giflib
+, gawk
+, makeBinaryWrapper
+, icoutils
+, copyDesktopItems
+, makeDesktopItem
+}:
+
+stdenv.mkDerivation rec {
+  pname = "maltego";
+  version = "4.6.0";
+  src = fetchzip {
+    url = "https://downloads.maltego.com/maltego-v4/linux/Maltego.v${version}.linux.zip";
+    hash = "sha256-q+1RYToZtBxAIDSiUWf3i/3GBBDwh6NWteHiK4VM1HY=";
+  };
+
+  postPatch = ''
+      substituteInPlace bin/maltego \
+            --replace /usr/bin/awk ${gawk}/bin/awk
+  '';
+
+    desktopItems = [
+    (makeDesktopItem {
+      name = pname;
+      desktopName = "Maltego";
+      exec = pname;
+      icon = pname;
+      comment = "An open source intelligence and forensics application";
+      categories = [ "Network" "Security" ];
+      startupNotify = false;
+    })
+  ];
+
+  nativeBuildInputs = [
+    icoutils
+    makeBinaryWrapper
+    copyDesktopItems
+  ];
+
+  buildInputs = [ jre giflib ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/{bin,share}
+    chmod +x bin/maltego
+    cp -aR . "$out/share/maltego/"
```

Comment:
are you sure that we need everything there, seems like *a lot*? I'm quite certain we can at least delete `bin/maltego` after doing the makeWrapper thing because that doesn't appear to be a symlink.